content/applications/finance/documents.rst

```diff
@@ -69,8 +84,8 @@ organized per category and filters can be used to sort them.
 Documents management
 ====================
 
-When clicking on a specific document, the right panel displays different options. On the top,
-additional options might be available: :guilabel:`Download`, :guilabel:`Share`, :guilabel:`Replace`,
+When selecting a document, the right panel displays different options. On the top, additional
```

Comment:
When opening a document, the right panel displays different options.

Reply:
selecting or opening